packages/hardhat/hardhat.config.ts

```diff
@@ -32,14 +32,15 @@ const config: HardhatUserConfig = {
       accounts: accounts(process.env.HARDHAT_FORK),
       forking: process.env.HARDHAT_FORK
         ? {
-            url: node_url(process.env.HARDHAT_FORK),
+            url: process.env.HARDHAT_FORK_URL ? process.env.HARDHAT_FORK_URL : node_url(process.env.HARDHAT_FORK),
```

Comment:
Don't you need to use node_url on HARDHAT_FORK_URL ? like `node_url(process.env.HARDHAT_FORK_URL)`

Reply:
Actually, your comment made me realize that I can just use the `HARDHAT_FORK` and create a new env called `ETH_NODE_URI_NETWORK` as specified in: https://github.com/1Hive/gardens/blob/master/packages/hardhat/helpers/network.ts#L15